### DISCUSSION TITLE
Issues with CoreWebView2 Settings Not Applying to New Windows in WebView2

### DISCUSSION THREAD

**Discussion by timepasser00** [MicrosoftEdge/WebView2Feedback#4643] [Q&A]
Hi everyone,

I'm encountering an issue with WebView2 where the CoreWebView2 settings do not apply in certain scenarios. Specifically, when I perform the following actions:

Click on "Share" and then select Twitter, a new window opens, but the CoreWebView2 settings are not applied to this window. Additionally, the new window requested event callback is not received.
Open a tool window, and again, the CoreWebView2 settings are not applied, and the new window requested event callback is not received.
Has anyone else experienced similar issues? How can I ensure that the CoreWebView2 settings are applied to these new windows and that the new window requested event callback is properly received?

Any help or suggestions would be greatly appreciated!

Thank you.

**Comment by DeTK** [comment]
I don't think there's a way to get an event that's been released yet
So the way I used it, I control the page with an extension in a separate way
If that's the best way right now, I think that's right

The embedded **webview.CoreWebView2.NewWindowRequested** event has not yet provided a good experience for hands-on experience

There might be some weird grammar, but I hope it's delivered well
